vmdk_plugin/sanity_test.go

```diff
@@ -240,7 +242,7 @@ func TestSanity(t *testing.T) {
 			"size": "1gb",
 		},
 	}
-	// Create/delete routine
+	// Create/delete goroutine
```

Comment:
I wasn't aware of that possibility, or the fact that esx would error out. But independently of skipping the tests for that specific scenario we should serialize reconfig tasks per vm in vmdk_ops.